File: apps/dotnet/Simex/Simex/Models/Incoterm.cs
using System;
using System.Collections.Generic;

namespace Simex.Models;

public partial class Incoterm
{
    public int Id { get; set; }

    public int IncotermTypeId { get; set; }

    public int TrackingStepId { get; set; }

    public virtual IncotermType IncotermType { get; set; } = null!;

    public virtual ICollection<Offer> Offers { get; set; } = new List<Offer>();

    public virtual ICollection<Operation> Operations { get; set; } = new List<Operation>();

    public virtual TrackingStep TrackingStep { get; set; } = null!;
}

File: apps/web/frontend/src/interfaces/incoterm/incoterm.ts
import type { IncotermType } from '@interfaces/incoterm/incotermType';
import type { TrackingStep } from '@interfaces/incoterm/trackingStep';

export interface Incoterm {
  incotermType: IncotermType | null;
  trackingStep: TrackingStep | null;
}
